minification.ipynb

```diff
@@ -0,0 +1,993 @@
+{
```

Comment:
<blockquote>You might be asking yourself: Is this "safe" to do? The answer lies in noticing that most post-training routines do not in fact require the full counts.</blockquote><p> </p><p>I would change this to say that "noticing that for many of the models in this codebase, post-training routines..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/scverse/scvi-tutorials/pull/98/discussion/#comment-1091363073'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>